src/sections/Meshery/Features-Col/index.js

```diff
@@ -42,7 +54,7 @@ const Features = () => {
                     <Counter
                       duration={6}
                       separator=","
-                      end={feature.count.value}
+                      end={feature.count.value !== null ? feature.count.value:performanceCount}
```

Comment:
Update this as well, as per the above comment.
Also, add improve code readability around `:` (else condition)